Base/Logic/Resources/ColorFiles/BlueRed.txt

```diff
@@ -0,0 +1,256 @@
+# Inspired form matplotlib colormaps: https://matplotlib.org/users/colormaps.html
```

Comment:
Thanks for reviewing.

For the ones I added in #582 the [licence](https://github.com/BIDS/colormap/blob/master/colormaps.py) is [CC0](https://creativecommons.org/share-your-work/public-domain/cc0), as they say [here](https://bids.github.io/colormap/).